rust/src/store.rs

```diff
@@ -62,10 +62,23 @@ impl IndexerStore {
         database_opts.create_missing_column_families(true);
         database_opts.create_if_missing(true);
 
-        let column_families: Vec<ColumnFamilyDescriptor> = Self::COLUMN_FAMILIES
+        let mut column_families: Vec<ColumnFamilyDescriptor> = Self::COLUMN_FAMILIES
             .iter()
             .map(|cf| ColumnFamilyDescriptor::new(*cf, cf_opts.clone()))
             .collect();
+
+        // Need to use a custom comparator for fee transfers
+        let mut feetransfer_opts = speedb::Options::default();
+        feetransfer_opts.set_compression_type(DBCompressionType::Zstd);
+        feetransfer_opts.create_missing_column_families(true);
+        feetransfer_opts.create_if_missing(true);
```

Comment:
Should we do `let mut feetransfer_opts = database_opts.clone()`?

Reply:
Yup, that would make it shorter